Utilities/StrFmt.cpp

```diff
@@ -286,6 +286,15 @@ struct fmt::cfmt_src
 
 		return 0;
 	}
+
+	static constexpr std::size_t size_char  = 1;
+	static constexpr std::size_t size_short = 2;
+	static constexpr std::size_t size_int   = 0;
```

Comment:
Thanks, I understand better now. So `int` is only special on pc? Btw, char/short could be done with `sizeof` too for consistency.